lib/ceased_member_row.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+require 'scraped'
+
+class CeasedMemberRow < Scraped::HTML
+  field :source do
+    noko.css('a @href').text
+  end
+
+  field :name do
+    noko.css('a').text
+  end
+
+  field :party do
+    # The layout is inconsistent. Scraping this way
+    # we always get the party name for each row
+    noko.xpath('text()')[0..1].text.split('.').last.tidy
```

Comment:
This is still odd on multiple fronts.  Calling `[0..1]` is always a bit of warning sign — that's usually simpler as `.take(2)`. But here you're calling `text` on the result, which is even more odd — especially as you're then calling `.split.last` to get rid of the first one again. 

So you could simplify all of this as:

```ruby
    noko.xpath('text()').take(2).last.text.tidy
```

However, that's probably not as good as being more expressive about what you actually want, which is the first text element after the linked name, which would be:

```ruby
    noko.xpath('a/following-sibling::text()').first.text.tidy
```